Workbooks/SapMonitor/PrometheusHaCluster/PrometheusHaCluster.workbook

```diff
@@ -403,7 +407,8 @@
                         },
                         {
                           "columnMatch": "value_d",
-                          "formatter": 5
+                          "formatter": 5,
+                          "formatOptions": {}
```

Comment:
Are all formatOptions() supposed to be null ? If yes, why use them?

Reply:
The formatOptions were automatically added by the workbook editor.  As far as I know, there are no impacts to leaving them in - would you propose us editing the automatically-generated workbook json to remove these?  If so, I'd suggest having some type of processor that removes everything that should be removed when putting the json for a workbook in GitHub..